senza/components/elastic_load_balancer_v2.py

```diff
@@ -96,6 +96,8 @@ def component_elastic_load_balancer_v2(definition,
     else:
         loadbalancer_subnet_map = "LoadBalancerSubnets"
 
+    vpc_id = configuration.get("VpcId") or account_info.VpcID
```

Comment:
actually `configuration.get("VpcId") or account_info.VpcID` is better as this will also work if VpcId is  false/empty